### PR TITLE
Add ./run command!

### PR DESCRIPTION
**Pull request by WillMoggridge**
- Add ./run command, which runs through Docker
- Add a polymer wrapper script to set a port if it is in the env vars
- Update documentation
- Removed `build` script for now as it auto ran with `./run` and complicated it.

## QA
- `./run` should start up example content
- `./run serve [FILE or FOLDER]` should serve a specific location
- It should run on port 8088 as standard, and show in console output

### CODE REVIEW

**Comment by WillMoggridge** [comment]
[Add ./run for Mac users \o/](https://trello.com/c/5inJiXGd/3291-add-run-for-mac-users-o)

**Comment by WillMoggridge** [comment]
Will update the `./run` docker image shortly

**Comment by WillMoggridge** [comment]
Waiting on this PR, working on it with @nottrobin https://github.com/canonical-webteam/generator-canonical-webteam/pull/16

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+PORT=8088
```
Comment (nottrobin) [inline .env]: How would you like to use `8016`? The next available - https://github.com/canonical-webteam/practices/pull/23/files.

File: README.md [hunk 2]
```diff
@@ -9,8 +9,48 @@ For technical details and help, go to [HACKING.md](HACKING.md)
...
+
+NPM/Yarn scripts:
+- `./run yarn serve-live`: Run local server with live content.
+- `./run yarn build-all`: Generate live tutorials and build live site.
+- `./run yarn build-site`: Build site to `build` folder.
+- `./run yarn build-tutorials`: Generate live tutorials.
+- `./run yarn polymer [args]`: Run a command through Polymer.
+
+
+### Running without Docker
```
Comment (nottrobin) [inline README.md]: Generally I prefer READMEs to be short and unassuming, which is why [www.ubuntu.com's](https://github.com/canonical-websites/www.ubuntu.com/blob/master/README.md) just says "For more detailed local development instructions, see HACKING.md." Is this something you'd like to emulate?

**Comment by WillMoggridge** [comment]
I made an upstream PR to sort out serve not getting arguments: https://github.com/canonical-webteam/generator-canonical-webteam/pull/19

File: .env (hunk 1 above)
Comment (WillMoggridge) [inline .env]: Done-diddily-done

File: README.md (hunk 2 above)
Comment (WillMoggridge) [inline README.md]: OK, I did similar here

File: package.json [hunk 3]
```diff
@@ -10,14 +10,15 @@
   },
   "homepage": "https://tutorials.ubuntu.com",
   "scripts": {
```
Comment (nottrobin) [inline package.json]: Could you add a `"clean"` script which deletes at least the "node_modules/" directory?

**Comment by WillMoggridge** [comment]
OK! I have updated to latest run and added a clean script. This should be sorted now!

**Pull request merged**